CHANGELOG.md

```diff
@@ -4,6 +4,55 @@
+
+The [jupyterlab-recents](https://github.com/jupyterlab-contrib/jupyterlab-recents) extension was integrated into JupyterLab.
+
+Users are now able to:
+
+- re-open recently used files from the main menu:
+
+   <img alt="An 'Open Recent' sub-menu in the 'File' menu allowing to re-open recently used files" src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-recent-submenu.png" class="jp-screenshot">
+- re-open recently closed files from the sidebar:
+
+   <img alt="A new 'Recently Closed' section in the 'Running' sidebar" src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-recent-sidebar.png" class="jp-screenshot">
+- quickly jump to open tabs/recently closed files by using a new searchable modal dialog (press <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>A</kbd> to open the dialog, or click on the \[↗\] icon in the sidebar):
+   <img alt="A modal dialog for quickly jumping to open/closed files" src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-recent-modal.png" class="jp-screenshot">
+
+### Full notebook windowing mode by default
+
+Notebooks in the `full` windowing mode only render the visible cells, significantly improving the performance of the application. The limitation of `full` mode is that the search function in your browser may produce false negatives, as discussed [here](https://jupyterlab.readthedocs.io/en/latest/user/interface.html#searching) - using the JupyterLab search function is recommended. To revert to the behaviour from JupyterLab 4.1 go to Settings → Settings Editor → Notebook, scroll to "Windowing mode" and choose `defer`.
+
+### Improved Shortcuts Editor
+
+Among numerous improvements and bug fixes for keyboard shortcuts editor:
+- it is now possible to remove the default shortcuts,
+- shortcuts are correctly sorted when using a language pack,
+- shortcuts with different arguments are now correctly displayed as individual entries.
+
+<img alt="A context menu opened over a keybinding in the Keyboard Shortcuts setting panel within the Settings widget" src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-delete-shortcut.png" class="jp-screenshot">
+
+### Dark high contrast theme
+
+A new dark high contrast theme, which is intended to benefit users with the need for higher contrast, following the WCAG AAA accessibility standard for color contrast.
+To select this theme choose from the menu bar: Settings → Theme → JupyterLab Dark High Contrast. Please do provide feedback and suggestions on further improvements to this theme.
+
+<img alt="A dark high contrast theme" src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-dark-high-contrast.png" class="jp-screenshot">
+
+### Keyboard shortcuts improvements
```

Comment:
The other headings are more specific so I tried to make this one more specific too

```suggestion
### JupyterLab-ify cell editor (CodeMirror) keyboard shortcuts
```

Reply:
This heading is meant to also be a place for a point on https://github.com/jupyterlab/jupyterlab/pull/15927 (once it is merged)

Reply:
I believe non-native speaker (most of the world) will struggle to understand `JupyterLab-ify`. Also, these shortcuts work the same in File editor and other editors (settings editor, metadata form, etc) so saying "cell editor" is non acurate.